quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ImportAtStartupDistTest.java

```diff
@@ -45,6 +46,23 @@ void testImport(LaunchResult result) {
         cliResult.assertMessage("Imported realm quickstart-realm from file");
     }
 
+    @Test
+    @BeforeStartDistribution(CreateRealmConfigurationFileAndDir.class)
+    @Launch({"start-dev", "--import-realm", "--log-level=org.keycloak.quarkus.runtime.storage.database.jpa:debug"})
+    void testImportAndIgnoreDirectory(LaunchResult result) {
```

Comment:
would you mind adding a test checking the functionality with symlinks?

Reply:
If not explicitly disabled why check for symlinks if we already have tests for files? For me, it is unnecessary. Or am I missing something?